lib/pcpg/monitor.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require 'zero_silent_failures/monitor'
+
+module PCPG
+  ##
+  # Monitor functions for Rails logging and StatsD
+  #
+  class Monitor < ::ZeroSilentFailures::Monitor
+    # statsd key for api
+    CLAIM_STATS_KEY = 'career-guidance-application'
+
+    # statsd key for submit career counseling sidekiq
+    SUBMISSION_STATS_KEY = 'worker.lighthouse.submit_career_counseling_job'
+    # statsd key for benefits intake sidekiq
+    BENEFITS_INTAKE_SUBMISSION_STATS_KEY = 'worker.lighthouse.submit_benefits_intake_claim'
+
+    def initialize
+      super('career-guidance-application')
+    end
+
+    def track_submission_exhaustion(msg, claim = nil)
```

Comment:
I don't see a test for the `claim = nil` condition. Can you write one if there isn't one?

Reply:
done!